src/allmydata/test/test_system.py

```diff
@@ -519,20 +520,26 @@ def _corrupt_mutable_share(self, filename, which):
         msf.writev( [(0, final_share)], None)
 
 
-    def test_mutable(self):
+    def test_mutable_sdmf(self):
+        return self._test_mutable(SDMF_VERSION)
+
+    def test_mutable_mdmf(self):
+        return self._test_mutable(MDMF_VERSION)
```

Comment:
docstrings would be nice here.

though, reading further, I see the single test being replaced tests approximately one zillion different things so ... :shrug:

Reply:
Yeah...